op-deployer/book/src/reference-guide/custom-deployments.md

```diff
@@ -0,0 +1,187 @@
+# Custom Deployments
+
+While OP Deployer was designed primarily for use with that are governed by Optimism, it also supports managing cusotm
+deployments. This is particularly common for RaaS providers, whose customers often request deployments with custom L1s
+(or L2s, in the case of L3s) or governance. This guide will walk you through the process of managing these chains using
+OP Deployer.
+
+> [!danger]
+> Chains deployed in this way are not subject to Optimism Governance. They may be running customized or unaudited
+> code. Use at your own risk.
+
+## Bootstrapping
+
+The first step to deploying a custom Superchain is to bootstrap it onto an L1. This process will:
+
+- Deploy contract implementations that will be shared among all OP Chains deployed using this Superchain.
+- Deploy Superchain-wide management contracts like `SuperchainConfig` and `SuperchainVersions`.
+- Set up ownership so that you can control the Superchain.
+
+You will use the `bootstrap` [family][bootstrap] of commands on `op-deployer` to do this.
+
+[bootstrap]: ../user-guide/bootstrap.md
+
+### Superchain Bootstrap
+
+Every OP Chain belongs to a logical Superchain. The Superchain consists of a set of contracts that control the behavior
+of a group of OP Chains. This includes:
+
+- Pausing bridges
+- Signaling which protocol versions are required and recommended
+
+You can deploy a new Superchain for each OP Chain, or you can deploy it once and share it across multiple OP Chains.
+The choice is up to the deployer. Note that you **cannot** share a Superchain across multiple L1s.
+
+To begin, bootstrap the Superchain onto your chosen L1 with the following command:
+
+```shell
+op-deployer bootstrap superchain \
+  --l1-rpc-url="<rpc url>" \
+  --private-key="<contract deployer private key>" \
+  --artifacts-locator="<locator>" \
+  --outfile="<path to outfile>" \
+  --superchain-proxy-admin-owner="<role address>" \
+  --protocol-versions-owner="<role address>" \
+  --guardian="<role address>"
+```
+
+This will output a JSON file containing the addresses of the relevant contracts. Keep track of this file, as you will
+need it in subsequent steps.
+
+We recommend the following these best practices when bootstrapping the Superchain:
+
+1. Use Gnosis SAFEs for ownership roles like `guardian` and `superchain-proxy-admin-owner`. The owner **must** be a
+   smart contract to support future upgrades, so a SAFE is a sensible default.
+2. Use a regular EOA as the deployer. It will not have any control over the Superchain once the deployment completes.
+3. Use a standard contracts tag (e.g. `tag://op-contracts/v2.0.0`). This will make upgrading easier.
```

Comment:
Should we nudge people to use the **latest** standard contracts tag here?

Reply:
folks can make that decision on their own IMO